web_ui/src/core/projects/services/in-memory-project-service.ts

```diff
@@ -375,7 +378,13 @@ export const createInMemoryProjectService = (): ProjectService => {
         };
     };
 
-    const exportProject = async (_datasetIdentifier: DatasetIdentifier): Promise<ProjectExport> => {
+    const exportProject = async ({
+        projectIdentifier,
+        selectedModelExportOption,
+    }: {
```

Comment:
```suggestion
    const exportProject = async (_: {
```

Can you try making this change and remove the eslint disable line?

Reply:
that worked! thanks Mark! :)